lib/puppet/version.rb

```diff
@@ -72,6 +72,10 @@ def self.version=(version)
     @puppet_version = version
   end
 
+  def self.major_minor_version
+    self.version.sub(/\A(\d+\.\d+)\..*$/, '\1')
+  end
```

Comment:
I changed one of the tests for the hiera function. It checks that the URL contains only major and minor version. I think that is sufficient to test this.